docs/_documentations/codewind-eclipse-quick-guide.md

```diff
@@ -0,0 +1,167 @@
+    * If you don't see an Appsody template, select the `Manage Template Sources...` link at the end of the window.
+    * Select the `Appsody Stacks - appsodyhub` checkbox. 
+    * Click `OK`.
+    * The templates are refreshed, and the Appsody templates are available. 
+5. Click `Finish`.
+    * To monitor your project's progress, right-click on your project, and select `Show Log Files`.
+    * Select `Show All`. Then a `Console` tab is displayed where you see your project's build logs. 
+
+Your project is displayed in the `Local [Running]` section. The progress for creating your project is tracked next to the project's name. 
+
+Your project is complete when you see your project is running and its build is successful. 
+
+### Accessing the application endpoint in a browser
+1. Return to your project under the Codewind tab. 
+2. Right-click your project and select `Open Application`. 
+
+Your application is now opened in the browser, showing the welcome to your Appsody microservice page. 
+
+### Adding a REST service to your application
+1. Go to your project's workspace under the Project Explorer tab. 
+2. Navigate to `Java Resources->src/main/java->dev.appsody.starter`. 
+3. Right-click `dev.appsody.starter` and select `New->Class`.
+4. Create a Class file, name it `Calculator.java`, and select `Finish`. This file is your JAX-RS resource. 
+5. Populate the file with the following code then **save** the file: 
+
+```
+package dev.appsody.starter;
+
+import javax.ws.rs.core.Application;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import javax.ws.rs.PathParam;
+
+@Path("/calculator")
+public class Calculator extends Application {
+
+    @GET
+    @Path("/aboutme")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String aboutme(){
+        return "You can add (+), subtract (-), and multiply (*) with this simple calculator.";
+    }
+
+    @GET
+    @Path("/{op}/{a}/{b}")
+    @Produces(MediaType.TEXT_PLAIN)
+    public Response calculate(@PathParam("op") String op, @PathParam("a") String a, @PathParam("b") String b)
+    {
+        int numA = Integer.parseInt(a);
+        int numB = Integer.parseInt(b);
+
+      switch(op)
+      {
+          case "+":
+              return Response.ok(a + "+" + b + "=" + (Integer.toString((numA + numB)))).build();
+
+          case "-":
+              return Response.ok(a + "-" + b + "=" + (Integer.toString((numA - numB)))).build();
+
+          case "*":
+              return Response.ok(a + "*" + b + "=" + (Integer.toString((numA * numB)))).build();
+
+          default:
+              return Response.ok("Invalid operation. Please Try again").build();
+      }
+    }
+}
+```
+Any changes you make to your code is automatically built and re-deployed by Codewind and viewed in your browser.
+
+### Working with the microservice
+You now can work with your calculator. 
```

Comment:
First mention of calculator, perhaps needs more introduction earlier, and more orientation here, maybe:
### Working with the example calculator microservice
You can now work with the example calculator microservice.